pkgs/development/libraries/mapnik/default.nix

```diff
@@ -83,9 +84,13 @@ stdenv.mkDerivation rec {
 
   cmakeFlags = [
     # Would require qt otherwise.
-    "-DBUILD_DEMO_VIEWER=OFF"
+    "-DBUILD_DEMO_VIEWER:BOOL=OFF"
+    "-DFONTS_INSTALL_DIR:PATH=lib/mapnik/fonts"
+    "-DPLUGINS_INSTALL_DIR:PATH=lib/mapnik/input"
```

Comment:
What made you add these two lines? Isn't this the default?

Reply:
I added a patch to address this behavior.